effect/jvm/src/main/scala/scalaz/effect/RTS.scala

```diff
@@ -128,6 +127,9 @@ trait RTS {
         future.cancel(true); ()
       }
     }
+
+  final def performed(canceler: PureCanceler): Canceler =
```

Comment:
I'd suggest calling this `impureCanceler` because it takes a `PureCanceler` and returns an impure one.

Reply:
brilliant, changing it